Rspec_tests/solver_spec.rb

```diff
@@ -0,0 +1,44 @@
+require './solver'
+
+describe Solver do
+  before :all do
+    @solver = Solver.new
+  end
+  context 'when given argument n to calculate factorial' do
+    it 'returns the factorial of the number given' do
+      expect(@solver.find_factorial(5)).to eql(120)
+    end
+
+    it 'returns 1 if argument is 0' do
+      expect(@solver.find_factorial(0)).to eql(1)
+    end
+
+    it 'raises Error when argument is negative integer' do
+      expect(@solver.find_factorial(-5)).to eql("can't work on negative integers")
+    end
```

Comment:
- Also, you can do that for a negative value.
![image](https://user-images.githubusercontent.com/38635158/164499855-7a2dd848-02c3-4678-a17a-a76eb5123ce9.png)